applications/RANSModellingApplication/custom_utilities/rans_variable_utils.h

```diff
@@ -0,0 +1,387 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya (https://github.com/sunethwarna)
+//
+//
+
+#if !defined(KRATOS_RANS_VARIABLE_UTILS)
+#define KRATOS_RANS_VARIABLE_UTILS
+
+/* System includes */
+
+/* External includes */
+
+/* Project includes */
+#include "includes/define.h"
+#include "includes/kratos_flags.h"
+#include "includes/model_part.h"
+#include "rans_modelling_application_variables.h"
+#include "utilities/openmp_utils.h"
+#include "utilities/variable_utils.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * @class RansVariableUtils
+ * @ingroup KratosRANSModellingApplication
+ * @brief This class extends @KratosCore VariableUtils class to perform some RANS unique
+ * tasks related with the variable values.
+ * @details The methods are exported to python in order to add this improvements to the python interface
+ * @author Suneth Warnakulasuriya
+ * @see VariableUtils
+ */
+class RansVariableUtils : public VariableUtils
```

Comment:
I would not derive...

Why not make it just a namespace? (This is what I would do with the `VariableUtils` if I would write them from scratch

also better to separate into header/source, esp if you include this in many places

Reply:
done